source/_partials/content/https-requirements.html

```diff
@@ -1,10 +1,8 @@
 <h3>Requirements for Automated Certificate Renewal</h3>
 <ul>
-  <li>All A/AAAA records must point to a server that serves Pantheon's acme-challenge response (e.g., <code>http://23.185.0.1/.well-known/acme-challenge/[variableDataHere]</code>).</li>
-  <li>Authority Name Servers must serve mixed-case lookups.</li>
-  <li>Authority Name Servers must not fail CAA lookups.</li>
+  <li>All A/AAAA/CNAME/DNAME DNS records must point to Pantheon's servers so Let's Encrypt can verify domain ownership.</li>
+  <li>Authoritative Name Servers must serve mixed-case lookups, and must not fail CAA lookups.</li>
   <li>CAA records must either 1) not exist for the domain and its parent domains or 2) authorize Let's Encrypt.</li>
-  <li>CNAMEs must follow the above requirements.</li>
-  <li>DNAMEs must not be used.</li>
+  <li>AAAA records are not required, but if set must exclusively point to Pantheon.</li>
```

Comment:
How about we keep and move it to bullet point 2?  @kf6nux   Alex and I discussed outside of this PR that it's probably a good idea to keep because there's some ambiguity around AAAA records being "required" from our UI.

Reply:
@ari-gold Do you mean merge it with the current second bullet point, or moving it into the 2nd position? I'm fine with the latter.

Reply:
The latter =)